Example/CollectionViewController.swift

```diff
@@ -79,12 +84,16 @@ extension CollectionViewController {
     static func viewModel(forState groups: [ToolGroup], onDeleteClosure: @escaping (Tool) -> Void) -> CollectionViewModel {
         let sections: [CollectionViewModel.SectionModel] = groups.map { group in
             let cellViewModels = group.tools.map { CollectionToolCellModel(tool: $0, onDeleteClosure: onDeleteClosure) }
-            return CollectionViewModel.SectionModel(
-                cellViewModels: cellViewModels,
-                headerHeight: 44,
-                footerHeight: 44,
-                diffingKey: group.name
+            let headerViewModel = CollectionViewHeaderModel(
+                title: group.name,
+                height: nil,
```

Comment:
wouldn't this fix your header size problem?

Reply:
Yikes - let me try that

Reply:
Yeah all fixed.

Reply:
lol